src/Fixers.php

```diff
@@ -72,143 +97,237 @@ class Fixers
         'phpdoc_summary',
         'phpdoc_to_comment',
         'phpdoc_trim',
         'phpdoc_type_to_var',
         'phpdoc_types',
         'phpdoc_var_to_type',
         'phpdoc_var_without_name',
-        'php_closing_tag',
         'pre_increment',
-        'php_unit_construct',
-        'php_unit_strict',
         'print_to_echo',
         'psr0',
         'psr4',
-        'remove_leading_slash_use',
-        'remove_lines_between_uses',
-        'return',
+        'random_api_migration',
         'self_accessor',
         'short_array_syntax',
-        'short_bool_cast',
-        'short_echo_tag',
-        'short_tag',
-        'single_array_no_trailing_comma',
+        'short_scalar_cast',
+        'simplified_null_return',
+        'single_blank_line_at_eof',
         'single_blank_line_before_namespace',
+        'single_import_per_statement',
         'single_line_after_imports',
         'single_quote',
-        'spaces_after_semicolon',
-        'spaces_before_semicolon',
-        'spaces_cast',
-        'standardize_not_equal',
-        'strict',
+        'space_after_semicolon',
+        'standardize_not_equals',
+        'strict_comparison',
         'strict_param',
-        'ternary_spaces',
-        'trailing_spaces',
+        'switch_case_semicolon_to_colon',
+        'switch_case_space',
+        'ternary_operator_spaces',
+        'trailing_comma_in_multiline_array',
         'trim_array_spaces',
         'unalign_double_arrow',
         'unalign_equals',
-        'unary_operators_spaces',
-        'unneeded_control_parentheses',
-        'unused_use',
-        'visibility',
-        'whitespacy_lines',
+        'unary_operator_spaces',
+        'unix_line_endings',
+        'visibility_required',
+        'whitespace_after_comma_in_array',
     );
 
-
     /**
      * @var string[]
      */
     public static $risky = array(
         'ereg_to_preg',
-        'php4_constructor',
+        'no_php4_constructor',
         'php_unit_construct',
+        'php_unit_dedicate_assert',
         'php_unit_strict',
         'psr0',
         'psr4',
-        'strict',
+        'strict_comparison',
         'strict_param',
     );
 
+    /**
+     * @var string[]
+     */
+    public static $aliases = array(
+        'operators_spaces'                           => 'binary_operator_spaces',
+        'line_after_namespace'                       => 'blank_line_after_namespace',
+        'blankline_after_open_tag'                   => 'blank_line_after_opening_tag',
+        'return'                                     => 'blank_line_before_return',
+        'spaces_cast'                                => 'cast_spaces',
+        'short_tag'                                  => 'full_opening_tag',
+        'newline_after_open_tag'                     => 'linebreak_after_opening_tag',
+        'alias_functions'                            => 'no_alias_functions',
+        'join_function'                              => 'no_alias_functions',
```

Comment:
Before it was `'join_function' => 'alias_functions'`.

Why it's changed @GrahamCampbell?